### PR TITLE
[12.0][FIX] base_custom_info

### PR DESCRIPTION
**Pull request by AdriaGForgeFlow**
Small fix on the creation of new templates:

image

cc ~ @ForgeFlow @pedrobaeza @Tardo

### CODE REVIEW

File: base_custom_info/models/custom_info_property.py [hunk 1]
```diff
@@ -83,20 +83,22 @@ def check_access_rule(self, operation):
         self.mapped("template_id").check_access_rule(operation)
         return super().check_access_rule(operation)
 
+    @api.multi
```
Comment (pedrobaeza) [inline base_custom_info/models/custom_info_property.py]: No need of putting this: it's the default.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
For not changing too much the diff, you can create a method `_check_default_value_one` with the same content, and call it from the `_check_default_value` iterating over recordset.

File: base_custom_info/models/custom_info_property.py (hunk 1 above)
Comment (pedrobaeza) [reply base_custom_info/models/custom_info_property.py]: This is still here

File: base_custom_info/models/custom_info_property.py [hunk 2]
```diff
@@ -98,6 +96,13 @@ def _check_default_value(self):
                     _("Default value %s cannot be converted to type %s.") %
                     (self.default_value, selection[self.field_type]))
 
+    @api.constrains("default_value", "field_type")
+    def _check_default_value(self):
+        """Ensure the default value is valid."""
+        for rec in self:
+            rec._check_default_value_one()
+
```
Comment (pedrobaeza) [inline base_custom_info/models/custom_info_property.py:104]: Extra empty line

Comment (AdriaGForgeFlow) [reply base_custom_info/models/custom_info_property.py:104]: Should be ok now :+1:

**Review by pedrobaeza — APPROVED** [review]
Let's fast-track this. Thanks for the patch

**Pull request merged**